### DISCUSSION TITLE
TF6760 Beckhoff Sample Code

### DISCUSSION THREAD

**Discussion by anahita2709** [Roald87/TcError#8] [Q&A]
Hi Roald,

I have to send Temperature Sensor value and System Time to the localhost server and the server only accepts JSON format.
Now in each 10 sec temperature value and the time retrieves form PLC and shows in JSON format by using TwinCAT3 .

For adding json header to the request I wrote following line of code before using SendJsonDomRequest method:

fbHeader.AddField(sField:='content-type',sValue:='application/json',FALSE);

HttpStatusCode returns 200 but when I want to Parse the server response I faced an issue and the result of jsonDoc shows 0

jsonDoc:=fbRequest.GetJsonDomContent(fbJson);

Flags status:
![image](https://github.com/Roald87/TcError/assets/135650133/92994156-4657-45d8-92aa-03c0b014fc57)

nContentSize =0 and fbJson =0

I would apprreciate it if you could please guide me on how I can fix this issue .

Thanks in advance.

Anahita

**Comment by Roald87** [comment]
Hi Anahita. This is not really the right place to post this question. You might want to try it at StackOverflow. I'm not sure how this issue could be fixed. I've never used HTTP requests with TwinCAT.
